tests/foreman/longrun/test_inc_updates.py

```diff
@@ -199,13 +189,18 @@ def setUp(self):
         self.addCleanup(vm_cleanup, self.vm)
         self.setup_vm(self.vm, rhel_6_partial_ak.name, self.org.label)
         self.vm.enable_repo(REPOS['rhva6']['id'])
+        timestamp = datetime.utcnow()
         self.vm.run('yum install -y {0}'.format(REAL_0_RH_PACKAGE))
 
-        # Find the content hosts and save them
-        hosts = Host(organization=str(self.org.id)).search()
-        self.partial_hosts = []
-        for host in hosts:
-            self.partial_hosts.append(host)
+        # Find the content host and ensure that tasks started by package
+        # installation has finished
+        host = entities.Host().search(
+            query={'search': 'name={}'.format(self.vm.hostname)})
+        wait_for_tasks(
+            query='label = Actions::Katello::Host::UploadPackageProfile'
+                  ' and resource_id = {}'
+                  ' and started_at >= {}'.format(host[0].id, timestamp)
```

Comment:
Will this play nice with using using user with other than UTC timezone settings? Just to make sure

Reply:
Yeap, it accepts date in UTC by default, you need to specify timezone directly to use non-UTC time.